_includes/codepens/casechange/index.js

```diff
@@ -0,0 +1,6 @@
+tinymce.init({
+  selector: "textarea",
```

Comment:
This should be `"textarea#casechange"`, otherwise it could try and override other areas on the page. We maybe should also consider adding this to the menu since it's not in the default menus, otherwise you'll only be able to access this via the toolbar button. eg:

```
  ...
  toolbar: 'casechange',
  menubar: 'tools',
  menu: {
    tools: { title: 'Tools', items: 'casechange' }
  }
  ...
```

Reply:
Fixed